.travis.yml

```diff
@@ -24,6 +24,10 @@ script:
     - pep257 ./test/functional/neutronless/esd/test_esd.py
     - flake8 --ignore=H304 ./test/functional/neutronless/esd/test_esd.py
     - pylint -E -d E1101 test/functional/neutronless/esd/test_esd.py
```

Comment:
I understand why you added these files to pep257, but I don't like the way it clutters up the travis file.  Could there not be a way to add this as a part of tox, and have it run as part of tox -e style?